client/src/pages/CreateProject.js

```diff
@@ -0,0 +1,309 @@
+  Grid,
+  Checkbox,
+  TextField,
+  Select,
+  MenuItem,
+  Chip,
+  FormControl,
+  InputLabel,
+  FormGroup,
+  FormControlLabel,
+  Button,
+} from "@material-ui/core";
+
+import { makeStyles } from "@material-ui/core/styles";
+
+import Navbar from "../components/Navbar";
+
+const useStyles = makeStyles((theme) => ({
+  header: {
+    padding: "2rem",
+    borderBottom: "2px",
+    borderBottomColor: theme.primary,
+  },
+  divider: {
+    backgroundColor: theme.primary,
+    height: "2px",
+    width: "10%",
+    margin: "auto",
+  },
+  subtitle: {
+    padding: "2rem",
+  },
+  form: {
+    width: "auto",
+  },
+  select: {
+    padding: "2px",
+  },
+  inputLabel: {
+    color: "#000",
+    marginLeft: "2em",
+    fontWeight: "bold",
+    position: "absolute",
+    top: "50%",
+    transform: "translateY(-50%)",
+    fontSize: 14,
+  },
+  marginTop: {
+    marginTop: "2rem",
+  },
+  button: {
+    backgroundColor: theme.primary,
+    margin: "2rem 0",
+    color: theme.bgcolor,
+    height: "3rem",
+    width: "60%",
+  },
+  chip: {
+    margin: "2px",
+  },
+}));
+
+function VerifyCheckboxes(props) {
+  const [checkboxes, setCheckboxes] = useState({
+    verifyAge: false,
+    verifyId: false,
+    verifyPayment: false,
+  });
+
+  useEffect(() => {
+    if (checkboxes.verifyAge && checkboxes.verifyId && checkboxes.verifyPayment)
+      props.verify();
+  }, [checkboxes, props.verify]);
+
+  const handleCheckboxes = (event) => {
+    setCheckboxes({
+      ...checkboxes,
+      [event.target.name]: event.target.checked,
+    });
+  };
+
+  return (
+    <FormGroup>
+      <FormControlLabel
+        control={
+          <Checkbox
+            checked={checkboxes.verifyAge}
+            onChange={handleCheckboxes}
+            name="verifyAge"
+            color="primary"
+          />
+        }
+        label="I'm at least 18 years old"
+      />
+
+      <FormControlLabel
+        control={
+          <Checkbox
+            checked={checkboxes.verifyId}
+            onChange={handleCheckboxes}
+            name="verifyId"
+            color="primary"
+          />
+        }
+        label="I can verify a bank account and government-issued ID"
+      />
+
+      <FormControlLabel
+        control={
+          <Checkbox
+            checked={checkboxes.verifyPayment}
+            onChange={handleCheckboxes}
+            name="verifyPayment"
+            color="primary"
+          />
+        }
+        label="I have a debit and/or credit card"
+      />
+    </FormGroup>
+  );
+}
+
+function CreateProject(props) {
+  const classes = useStyles();
+
+  // State variables
+  const [industries, setIndustries] = useState([]);
+  const [validIndustries, setValidIndustries] = useState([]);
+  const [verified, setVerified] = useState(false);
+
+  // Populate valid industries
+  useEffect(() => {
+    axios.get("/api/v1/industries").then((res) => {
+      setValidIndustries(res.data);
+    });
+  }, [false]); // Only call on initial mount
```

Comment:
```suggestion
  }, []); // Only call on initial mount
```

an empty array will suffice and will only call on init